src/main/java/com/notably/model/block/BlockModel.java

```diff
@@ -0,0 +1,61 @@
+package com.notably.model.block;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * API of the BlockModel component.
+ */
+public interface BlockModel {
```

Comment:
Is this supposed to be `public interface Block`?

Reply:
I intended to have the implementing class be `Block`. What do you think?

Reply:
Fine with me! I'll change to match the naming convention!